lib/xclarity_client/client.rb

```diff
@@ -122,6 +126,14 @@ def fetch_switches(uuids = nil,
                                                    excludeAttributes)
     end
 
+    def fetch_storages(uuids = nil,
+      includeAttributes = nil,
```

Comment:
__Required:__
Please, use snake_case on attributes name

Reply:
Thanks for the reminder, I updated this attributes to snake_case.